src/components/Highlights.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="highlightsbar">
+    <span id="center-highlight">
+      {{ industryindices[highlightsindex1] }}
+      {{ industrybetas[highlightsindex1] }}
+    </span>
+    <span id="left-highlight"> JSE Allshare Index: {{ jseindex }}</span>
+    <span id="right-highlight">
+      {{ industryindices[highlightsindex2] }}
+      {{ industrybetas[highlightsindex2] }}
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Highlights',
+  data() {
+    return {
+      industrybetas: ['0.7276 (5%)', '0.9 (-7%)', '1.1 (25%)', '0.68 (-30%)'],
+      industryindices: [
+        'Oil and Gas Index: ',
+        'Basic Materials Index: ',
+        'Industrials Index: ',
+        'Consumer Goods Index: ',
+      ],
+      jseindex: '0.74 (+5%)',
+      highlightsindex1: 1,
+      highlightsindex2: 2,
+    }
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+#right-highlight {
+  float: right;
+  text-transform: uppercase;
```

Comment:
whenever you are repeating things it should sound an alarm. You can combine the text-transform and font-weight into 1 class and share that class with all the highlights therefore reducing the amount of code as well as making it a lot easier to change next time (we would only need to do it in one place instead of three)